Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -84,12 +86,13 @@ open class OptimizelyClient: NSObject {
                 odpManager: OdpManager? = nil,
                 defaultLogLevel: OptimizelyLogLevel? = nil,
                 defaultDecideOptions: [OptimizelyDecideOption]? = nil,
-                settings: OptimizelySdkSettings? = nil) {
+                settings: OptimizelySdkSettings? = nil,
+                clientName: String? = nil) {
```

Comment:
Can we move this new option into `OptimizelySdkSettings`? It's not a common configuration usage.

Reply:
We also need to config "sdkVersion" as well.
We can use "sdkName" and "sdkVersion" to be consistent with android-sdk?